src/coreclr/vm/clrex.cpp

```diff
@@ -1954,7 +1954,12 @@ OBJECTREF EECOMException::CreateThrowable()
         {
             // We have a non 0 help context so use it to form the help link.
             SString strMessage;
-            strMessage.Printf(W("%s#%d"), m_ED.bstrHelpFile, m_ED.dwHelpContext);
+            strMessage.Append(m_ED.bstrHelpFile);
+
+            // Add the help context
+            WCHAR cxt[ARRAY_SIZE("#") + MaxSigned32BitDecString] = W("#");
```

Comment:
Does the buffer size here and in the other uses of `FormatInteger` need `+1` for the terminating null?

Reply:
It doesn't for this one because it is included in `ARRAY_SIZE("#")`, but when it is one of the constants, it does. This is another justification for making a smart helper.